src/w2grid.js

```diff
@@ -3706,10 +3731,15 @@
                     if (this.records.length <= end + 1 || this.records[end+1].w2ui == null ||
                         stops.indexOf(this.records[end+1].w2ui.parent_recid) >= 0)
                         break;
-                    if (this.records[end+1].w2ui.expanded) this.records[end+1].w2ui.expanded = false;
                     end++;
                 }
                 this.records.splice(start, end - start + 1);
+                this.total -= end - start + 1;
+                var url = (typeof this.url != 'object' ? this.url : this.url.get);
+                if (!url) {
+                    if (this.searchData.length > 0)
+                        this.localSearch(true);
```

Comment:
The reason I don't like putting "if (condition) do_something" on a single line is that it makes debugging painful. By putting in on two lines: (1) I can put a breakpoint on the second line and (2) when I run step-by-step, I know if the condition has been taken or not.

My guess is that the "if (condition) do_something" style became common in javascript, (unlike C, C++, Java), because there was no good debugger in javascript until recently...

Ok, I've pushed another commit to add braces everywhere. Is that ok?